### PR TITLE
Added Functionality to NPCs Bhagirath and Sahn (On Ships to Selbina/Maura). Added Functionality to Airship Time NPCs and Randomized Player Starting Point on Airships.

### PR DESCRIPTION
**Pull request by JoeCT**
Added functionality to NPCs Bhagirath and Sahn.  These NPCs are the captains of the ships bound for either Mhaura or Selbina.  They now accurately relay approximate times to destination.

### CODE REVIEW

**Comment by teschnei** [comment]
space before brackets, not after! (yes, i'm anal about this :p)

File: scripts/zones/Ship_bound_for_Selbina/npcs/Bhagirath.lua [hunk 1]
```diff
@@ -21,7 +21,33 @@ end;
...
-	player:messageSpecial(ON_WAY_TO_SELBINA,0,0); -- Earth Time, Vana Hours. Needs a get-time function for boat?
+
+	local vHour = VanadielHour();
+	local vMin  = VanadielMinute();
+
+	while vHour >= 6 do
+		vHour = vHour - 8;
+	end
+
+	if(     vHour == -2) then vHour = 8;
```
Comment (TeoTwawki) [inline scripts/zones/Ship_bound_for_Selbina/npcs/Bhagirath.lua]: Are you trying to vertically align your var names or "then"s or something?
```lua
if (vHour == -2) then
    vHour = 8;
elseif (vHour == -1) then
    vHour = 7;
elseif (vHour == 0) then
    vHour = 6;
elseif (vHour == 1) then
    vHour = 5;
elseif (vHour == 2) then
    vHour = 4;
elseif (vHour == 3) then
    vHour = 3;
elseif (vHour == 4) then
    vHour = 2;
elseif (vHour == 5) then
    vHour = 1;
end
```
This is the style usually used, though not strictly enforced (I wish it were).

**Comment by JoeCT** [comment]
Updated both scripts.  Should be more to your liking.  - A while back I used to work as a programmer and we were evaluated based on lines of code written.  They had very strict guidelines concerning style  which is where I picked up the majority of mine.  Anyway, we were required to condense simple conditionals like the ones in these scripts down to as few lines as possible.  If not, the extra lines were actually counted against us.  Personally, I don't mind which style I use.  Just gotta break my old habits.

**Comment by JoeCT** [comment]
Added functionality to NPCs Dereck and Michele. These NPCs are on the Bastok-Jeuno Airship. They now accurately relay approximate times to destination.

**Pull request merged**